### DISCUSSION TITLE
What happens if you set ECUT less than 1 keV?

### DISCUSSION THREAD

**Discussion by rtownson** [nrc-cnrc/EGSnrc#827] [Physics] [answered]
I ask because we typically set `ECUT=0.512 MeV` to get a 1 keV cutoff, but the electron rest mass is larger than exactly 0.511 MeV. Is there an extrapolation on the data tables? Or is the smallest value in the table used when energy drops below 1 keV?

**Comment by Jaap1957** [comment, marked as answer]
The code uses:
E = 0.5109989461 MeV + kinetic_E
Wiki says : 0.51099895000 MeV rest mass

So the rest mass is actually a little below 0.511

**Reply by mchamberland** [reply]
@rtownson looks like there were new CODATA values published in 2018. Time to update the rest mass?

**Reply by ftessier** [reply]
Yes, indeed we should update to the most recent [CODATA 2018 recommended value](https://physics.nist.gov/cgi-bin/cuu/Category?view=html&Atomic+and+nuclear.x=105&Atomic+and+nuclear.y=14): `0.510 998 950 00(15) MeV`.

**Reply by rtownson** [reply]
Feels like we just did this! Yes we should take a look.
